homepage.css

```diff
@@ -95,7 +94,7 @@ ul li:hover ul{
 }
 
 .footer{
-	background-color: black;
+	background-color: blue;
```

Comment:
I change the background color for footer to blue (place holder, test to see if it work)